FoodFriends/app/src/main/java/com/example/foodfriends/fragments/ExploreFragment.java

```diff
@@ -47,12 +48,14 @@
 public class ExploreFragment extends Fragment {
```

Comment:
Now that we functionality for fetching and there's a mental model of how UI and business logic should flow together. We can probably look ahead and start using livedata objects to serve UI functionality. 

https://developer.android.com/topic/libraries/architecture/livedata

Reply:
Implement Live Data Models